submissions/busolaAdedoyin/javascript/loop.js

```diff
@@ -0,0 +1,17 @@
+var x = 123 -7;
+var i;
+
```

Comment:
@elizabethbusola Well done! I think it is best if `i` was declared in the condition for the for loop instead.

Reply:
@ToluAkin I have made the changes